src/SWP/Bundle/ContentBundle/Tests/Functional/RouteLoaderTest.php

```diff
@@ -25,6 +27,12 @@ class RouteLoaderTest extends WebTestCase
     public function setUp()
     {
         self::bootKernel();
+
+        $this->loadFixtures(
+            [
+                'SWP\Bundle\ContentBundle\Tests\Functional\app\Resources\fixtures\LoadArticlesData',
+            ], 'default'
```

Comment:
the second argument is invalid, as this function accept the second parameter of type `bool`. It can be removed. You can also put `[LoadArticlesData::class]` as a first argument